mesonbuild/backend/vs2010backend.py

```diff
@@ -327,7 +329,21 @@ def generate_solution(self, sln_filename, projlist):
                 for dep, target in recursive_deps.items():
                     if prj[0] in default_projlist:
                         default_projlist[dep] = target
-
+                    if prj[0] in tests_projlist:
+                        tests_projlist[dep] = target
+
+            buildtest_line = prj_templ % (self.environment.coredata.lang_guids['default'],
+                                     'BUILD_TESTS', 'BUILD_TESTS.vcxproj',
```

Comment:
__[Flake8]__ [E128] continuation line under-indented for visual indent ([link](https://sider.review/gh/repos/19784232/pulls/5728#issue-4434545))

<sub>posted by Sider</sub>